src/pages/guides/project/tools-libraries/index.md

```diff
@@ -4,51 +4,26 @@ title: External tools and libraries
 
 # External tools and libraries
 
-![tools and technology overview](images/pwa-tools-technology-overview.png)
-
-To work with the utilities provided by the PWA Studio project, you should be familiar with the tools and libraries listed in the following table:
-
-|                           |  Contributor  | Frontend developer | Sysadmin/Devops/Backend developer |
-| ------------------------- | :-----------: | :----------------: | :-------------------------------: |
-| [PHP](#php)               |               |                    |           _Recommended_           |
-| [JavaScript](#javascript) | **Required**  |    **Required**    |           _Recommended_           |
-| [webpack](#webpack)       | **Required**  |   _Recommended_    |                                   |
-| [React](#react)           | **Required**  |    **Required**    |                                   |
-| [Redux](#redux)           | _Recommended_ |   _Recommended_    |                                   |
-| [GraphQL](#graphql)       | **Required**  |    **Required**    |           **Required**            |
-| [Workbox](#workbox)       | **Required**  |    **Required**    |                                   |
+The following diagrams describe the route a request takes through PWA Studio and the technologies involved at each step.
+This topic provides an overview to some of the major technologies used and other resources.
```

Comment:
The deleted table still seems relevant. Why was it deleted?

Reply:
It was basically an overwrought TOC for the page and I didn't think it was adding much.